src/main/java/fi/aalto/cs/apluscourses/intellij/services/PluginSettings.java

```diff
@@ -239,4 +244,19 @@ public void startRegularSubmissionResultsPolling(@NotNull Project project) {
       regularSubmissionResultsPollingStarted = true;
     }
   }
+
+  /**
+   * Method that adds a new file (pattern) to the list of files not being shown in the Project UI.
+   *
+   * @param ignoredFileName a {@link String} name of the file to be ignored.
+   * @param project a {@link Project} to ignore the file from.
+   */
+  public static void ignoreFileInProjectView(@NotNull String ignoredFileName,
+                                      @NotNull Project project) {
+    FileTypeManager fileTypeManager = FileTypeManager.getInstance();
+    String ignoredFilesList = fileTypeManager.getIgnoredFilesList();
+    Runnable runnable = () -> fileTypeManager
+        .setIgnoredFilesList(ignoredFilesList + ignoredFileName + ";");
+    WriteCommandAction.runWriteCommandAction(project, runnable);
+  }
```

Comment:
This causes no errors even if the file doesn't exist, right?

Reply:
AFAIK, nope, it's just a name (like a pattern) that is not shown in the project view, if it can't be found, then nothing happens. I'm certain not all the project contain those files that are listed there by default, but it still checks.